sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/ServiceBusMessageBatch.cs

```diff
@@ -9,8 +9,11 @@
 namespace Azure.Messaging.ServiceBus
 {
     /// <summary>
-    ///   A set of <see cref="ServiceBusMessageBatch" /> with size constraints known up-front,
+    ///   A set of <see cref="ServiceBusMessage" /> with size constraints known up-front,
     ///   intended to be sent to the Queue/Topic as a single batch.
+    ///   A <see cref="ServiceBusMessageBatch"/> can be created using
+    ///   <see cref="ServiceBusSender.CreateBatchAsync(System.Threading.CancellationToken)"/>.
+    ///   Messages can be added to the batch using the <see cref="TryAdd"/> method on the batch.
```

Comment:
Nice!!

Reply:
Can we add this `Messages can be added to the batch using the <see cref="TryAdd"/> method on the batch` for `CreateBatchAsync` docs as well?